### PR TITLE
Add a case to illustrate how to use `kubectl get -o custom-columns`

### PR DESCRIPTION
**Pull request by WanLinghao**
/kind documentation

**What this PR does / why we need it**:
Add a case to illustrate how to use `kubectl get -o custom-columns`

```release-note
NONE
```

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
/kind documentation
/priority backlog

File: pkg/kubectl/cmd/get/get.go [hunk 1]
```diff
@@ -118,6 +118,9 @@ var (
 		# Return only the phase value of the specified pod.
 		kubectl get -o template pod/web-pod-13je7 --template={{.status.phase}}
 
+		# List resource information in custom columns.
+		kubectl get pod test-pod -o custom-columns=CONTAINER:.spec.containers[-2:].name,IMAGE:.spec.containers[-2:].image
```
Comment (neolit123) [inline pkg/kubectl/cmd/get/get.go]: this seems a little verbose / edge-case, but will leave it to the maintainers to comment.

Comment (WanLinghao) [reply pkg/kubectl/cmd/get/get.go]: Hi，I want to show this edge–case to let people, especially new users， to grab it thoroughly

**Comment by WanLinghao** [comment]
@liggitt PTAL

File: pkg/kubectl/cmd/get/get.go (hunk 1 above)
Comment (liggitt) [reply pkg/kubectl/cmd/get/get.go]: a custom-columns example is fine, I'd probably choose a simpler example jsonpath

**Comment by WanLinghao** [comment]
@liggitt Done,PTAL

**Pull request merged**